internal/ingress/status/status.go

```diff
@@ -559,6 +559,7 @@ func (s *statusSync) runUpdateTCPIngress(ctx context.Context,
 			return nil, nil
 		}
 
+		fmt.Println("runUpdateTCPIngress heheheh")
```

Comment:
This looks like a debugging line we need to remove:

```suggestion
```

Reply:
Good cache.